frontend/src/service/StaticConfigDecision.service.ts

```diff
@@ -1,6 +1,9 @@
 import { IFeatureDecisionService } from './FeatureDecision.service';
 
 export class StaticConfigDecisionService implements IFeatureDecisionService {
+  includeViewChangeLogButton(): boolean {
+    return false;
```

Comment:
You can read about feature toggle here to understand the reasoning behind our architecture: http://s/ft